oozie/src/main/java/org/apache/falcon/oozie/ImportExportCommon.java

```diff
@@ -129,4 +129,45 @@ public static StringBuilder buildTableArg(StringBuilder builder, String tableNam
         return builder.append("--table").append(ImportExportCommon.ARG_SEPARATOR)
                 .append(tableName);
     }
+
+    public static StringBuilder buildArguments(StringBuilder builder, Map<String, String> extraArgs)
+            throws FalconException {
+        handleVerbose(builder, extraArgs);
+        for(Map.Entry<String, String> e : extraArgs.entrySet()) {
+            if (!e.getKey().startsWith("--direct")) {
+                builder.append(e.getKey()).append(ImportExportCommon.ARG_SEPARATOR).append(e.getValue())
+                        .append(ImportExportCommon.ARG_SEPARATOR);
+            }
+        }
+        return builder;
+    }
+
+    public static StringBuilder handleVerbose(StringBuilder builder, Map<String, String> extraArgs)
+            throws FalconException {
+        if ((extraArgs.containsKey("--verbose")) && (extraArgs.get("--verbose").equalsIgnoreCase("TRUE"))) {
+            builder.append("--verbose").append(ImportExportCommon.ARG_SEPARATOR);
+            extraArgs.remove("--verbose");
+        }
+        return builder;
```

Comment:
return type is not required here as we are using the same builder received in the input.